content/docs/rfcs/14/README.md

```diff
@@ -132,6 +132,28 @@ If an optional attribute, such as `meta`, is absent, the concatenation of attrib
 
 This hashing schema is deemed appropriate for use cases where a cross-implementation deterministic hash is needed, such as message deduplication and integrity validation. The collision probability offered by this hashing schema can be considered negligible. This is due to the deterministic concatenation order of the message attributes, coupled with using a SHA-2 (256-bit) hashing algorithm.
 
+### Test vectors
+
+Waku message hash computation:
+```
+pubsub_topic = 0x2f77616b752f322f64656661756c742d77616b752f70726f746f
```

Comment:
pubsub topic and content topic are string type. I think it'd be worth to include the string value as I'd expect some implementations to work directly from the string type to the hash result.

payload and meta being byte array, hex format is appropriate :+1: